### ISSUE TITLE
Polygon Representation

### ISSUE THREAD

**Issue by ashnair1** [microsoft/torchgeo#121]
How does `torchgeo` represent targets that are polygons? I ask because this will be required for adding datasets like Spacenet or DOTA where annotations are polygons/polygon coordinates.

**Comment by adamjstewart** [comment]
Since TorchGeo is mostly designed for computer vision problems, in datasets that include polygon masks or object bounds, we immediately rasterize those bounds to images. See the `CanadianBuildingFootprints` dataset as an example of this.

**Comment by ashnair1** [comment]
I see, so the datasets internally are all images and masks. Does the scope of `torchgeo` include object detection (drawing per object bounding box) and instance segmentation (drawing per object mask)?

**Comment by adamjstewart** [comment]
Yes, definitely. The NWPU VHR-10 dataset includes both object detection bounding boxes and instance segmentation masks.
